app/models/sign_in/session_container.rb

```diff
@@ -41,5 +41,22 @@ def initialize(session:, # rubocop:disable Metrics/ParameterLists
     def persisted?
       false
     end
+
+    def context
```

Comment:
This will be used in the 'refresh' endpoint, so I'd remove a ton of these fields:
Remove: `access_token_uuid`, `access_token_audience`, `version`, `last_regeneration_time`, `access_token_Created_time`, `access_token_expiration_time`